apps/challenges/aws_utils.py

```diff
@@ -802,12 +802,21 @@ def create_eks_cluster(sender, challenge, field_name, **kwargs):
             config_text = yaml.dump(cluster_config, default_flow_style=False)
             config_file = NamedTemporaryFile(delete=True)
             config_file.write(config_text.encode())
-            ChallengeEvaluationCluster.objects.create(
-                challenge=challenge,
-                name=cluster_name,
-                cluster_endpoint=cluster_ep,
-                cluster_ssl=cluster_cert,
-            )
+            try:
+                ChallengeEvaluationCluster.objects.create(
```

Comment:
@Ayukha Let's try to get the object first and if it is not found then please create the object by serializing the data using a serializer. Also, the update should happen using the serializer as well.